extensions/sequelize/src/__tests__/integration/repository.integration.ts

```diff
@@ -179,6 +179,27 @@ describe('Sequelize CRUD Repository (integration)', () => {
       expect(afterResponse).to.not.have.property('password');
     });
 
+    it('[find] excludes undefined props from where filter', async () => {
+      const user = getDummyUser();
+      await userRepo.create({
+        name: user.name,
+        address: user.address as AnyObject,
+        email: user.email,
+        password: user.password,
+        dob: user.dob,
+        active: user.active,
+      });
+
+      const userData = await userRepo.find({
+        where: {name: user.name, email: user.email, password: undefined},
```

Comment:
Can you please add a comment explaining how is this excluding undefined password ? Because what I see above is you have assigned password as user.password. So I am not very sure what's the test case doing here ?